libs/shared/ui/src/lib/components/table/table-row-deployment/table-row-deployment.tsx

```diff
@@ -4,7 +4,7 @@ import {
   type DeploymentHistoryHelmResponse,
 } from 'qovery-typescript-axios'
 import { type MouseEvent, useState } from 'react'
-import { Link, useNavigate, useParams } from 'react-router-dom'
+import { Link, useParams } from 'react-router-dom'
```

Comment:
Rename this one RouterLink and let our implementation imported as Link

Reply:
https://github.com/Qovery/console/pull/1322/commits/9ea6fe2e4c8f1c42ca7802687dd221902a525bd1